api/src/test/java/com/epam/pipeline/test/web/AbstractControllerTest.java

```diff
@@ -81,13 +83,16 @@ protected final JsonMapper getObjectMapper() {
                                    final JsonMapper objectMapper,
                                    final T payload,
                                    final TypeReference<Result<T>> typeReference) throws Exception {
+        final ObjectMapper deserializationMapper = JsonMapper.newInstance();
+        deserializationMapper.disable(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES);
         final ResponseResult<T> expectedResult = buildExpectedResult(payload);
 
         final String actual = mvcResult.getResponse().getContentAsString();
         Assert.assertTrue(StringUtils.isNotBlank(actual));
         assertThat(actual).isEqualToIgnoringWhitespace(objectMapper.writeValueAsString(expectedResult));
 
-        final Result<T> actualResult = JsonMapper.parseData(actual, typeReference);
+        final Result<T> actualResult = JsonMapper.parseData(actual, typeReference, deserializationMapper);
+        assertNotNull(actualResult);
         Assert.assertEquals(expectedResult.getPayload(), actualResult.getPayload());
```

Comment:
Please use either `Assert.assertSomething` or just `assertSomething` syntax, not both. It is pretty hard to read if the code is not consistent.